routing_common/transit_types.hpp

```diff
@@ -87,34 +89,54 @@ class Stop
   // and deserialization.
 };
 
+class SingleMwmSegment
+{
+public:
+  SingleMwmSegment() = default;
+  SingleMwmSegment(Segment const & segment);
+
+  FeatureId GetFeatureId() const { return m_featureId; }
```

Comment:
How mwm id is used? What if we have two segments from different mwms with the same feature id?

Reply:
SingleMwmSegment is a class which represent segments at a certain mwm. So for transit it's necessary to same information about pedestrian feature in a mwm. It's assumed that mwm id of the segment is mwm id of mwm where the segment is saved.

I'd like to draw your attention that FeatureId is declared as uint32_t. So it's not FeatureID.

Did I answer you question?